src/components/CartItem.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useCart } from '../context/cart-context';
+
+export default function CartItem({ item }) {
+  const { image, productName, name, price, seller, count } = item;
+  const { dispatch } = useCart();
+  return (
+    <section class="cart__card-container box-shadow-right">
+      <div class="cart__img-div">
+        <img src={image} alt={productName} class="cart-img" />
+      </div>
+      <div class="cart__text-div p-6">
+        <h2>{name}</h2>
+        <p className="l-h-3">
+          <span className="price f-7">Rs. {parseInt(price)}</span>
```

Comment:
You shouldn't use parseInt here as the ideal purpose is to extract number out of a string.
Number() should be used for conversion into integers.
> Extracting and Converting are distinct things.